build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -425,6 +425,19 @@ message Action {
   bool do_not_cache = 7;
 
   reserved 8; // Used for field moved to [Command][build.bazel.remote.execution.v2.Command].
+
+  // An optional additional salt value used to place this `Action` into a
+  // separate cache universe or "epoch" from other instances having the
+  // same field contents. This salt typically comes from operational
+  // configuration specific to sources such as repo and service configuration,
+  // and allows disowning an entire set of ActionResults that might have been
+  // poisoned by buggy software or tool failures.
+  //
+  // As an example for repo configuration, a repo could contain a special file
```

Comment:
I would generally shy away from such a specific example in the proto comments. In this particular I think describing the simple case of incrementing an integer makes sense, but I probably wouldn't connect it specifically with how to add that integer to the repository.